lib/src/arithmetic/float_fixed.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// floating_fixed.dart
+// Transform floating point input into fixed point output.
+//
+// 2024 September 25
+// Author: Soner Yaldiz <soner.yaldiz@intel.com>
+
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+
+/// [FloatToFixedConverter] converts a floating point input to a signed
+/// fixed-point output following Q notation (Qm.n format) as introduced by
+/// (Texas Instruments)[https://www.ti.com/lit/ug/spru565b/spru565b.pdf].
+/// Infinities and NaN's are not supported. Conversion is lossless.
+/// The output is in two's complement and in Qm.n format where:
+/// m = 1 + (e_max - bias)
```

Comment:
Note that the doc comments will get squished into one line if written like this.  It's treated like markdown, so you need empty lines (for example) to force new-lines, or you can wrap things in triple ticks ``` to make a code block, or you can use bullet points with `-` or `*`